src/SignUpSheetAdvance.js

```diff
@@ -40,6 +40,7 @@ function clearOutCurrentSignUp(signUpSheet, signUpTemplate) {
 }
 
 function deleteCurrentSignupSection(signUpSheet, signUpTemplate) {
+  // TODO: remove extra columns if they aren't being used for anything
```

Comment:
So we are using fullName in the agenda generator - https://github.com/Mountain-View-Toastmasters/sign-up-sheet/blob/3088c9e36928673850cf5eea5e5366e610cfc709/src/AgendaMinutesGenerator.js#L87

And I thought there were some planned (but not yet realized) uses for email address e.g. generate the weekly confirmation email automatically.

Reply:
The `memberMap` is generated directly from the membership sheet though, which means as long as we have the short name, we can always get the related information like email/fullName without having to carry the details everywhere else.

Reply:
I think there's some convenience to already having the email address and full name available, but I doubt most people are using the hidden columns to get them so it's probably okay to drop these.

If these columns are really missing, are lines 44 and 45 even doing anything?

Reply:
It's not that they aren't convenient, its that they don't even exist in the first place (from what I can tell). When I manually hide columns, there's an indicator in the UI that shows there are hidden columns, but I don't see anything in the current sheet.

Reply:
@acmiyaguchi Just to note they definitely existed at some point. I remember being slightly annoyed with having to remove the columns every time I copy the list of members to send an email when I've been Toastmaster.

I suspect someone may have deleted the hidden columns sometime between now and the last time I was TM.